### ISSUE TITLE
Database migration unexpectedly failed

### ISSUE THREAD

**Issue by sant9ga** [ethcore/parity#2409]
I've updated parity from 1.3.1 to 1.3.2. It starts to migrate database from 9 to 10, but after 3-5 min it closed with the provided error. (win 8.1 x64), used parity in --geth mode. Now it doesn't work neither in --geth, neither in default mode.

**Comment by NikVolf** [comment]
can you check that there is enough space on disk
migration is requires at least the same amount of extra disk space that the previous database occupied

**Comment by sant9ga** [comment]
@NikVolf you're right! Maybe it would be great to add some error-notification about not-enought-free space?

**Comment by NikVolf** [comment]
#2424

**Comment by ferni** [comment]
Where temp files should I delete after the migration fails like this?

**Comment by Sjors** [comment]
Is there any way to tell how much space that is?

I had to close my laptop after about 5 minutes of syncing. When I reopened it a few hours later, I kept getting this error:
<img width="570" alt="schermafbeelding 2016-10-07 om 14 40 26" src="https://cloud.githubusercontent.com/assets/10217/19180930/106004b6-8c9c-11e6-88d5-89ca9153b792.png">

`version Parity/v1.3.4-beta-a618dca-20161006/x86_64-macos/rustc1.12.0`. I have 35.9 GB available.
